src/reformatters/noaa/hrrr/forecast_48_hour/template_config.py

```diff
@@ -345,399 +229,4 @@ def data_vars(self) -> Sequence[NoaaHrrrDataVar]:
-                ),
-                internal_attrs=NoaaHrrrInternalAttrs(
-                    grib_element="HGT",
-                    grib_description='0[-] CEIL="Cloud ceiling"',
-                    grib_index_level="cloud ceiling",
-                    index_position=117,
-                    keep_mantissa_bits=8,
-                    hrrr_file_type="sfc",
-                ),
-            ),
-            # HRRR provides 80m but not 100m winds
-            NoaaHrrrDataVar(
-                name="wind_u_80m",
-                encoding=encoding_float32_default,
-                attrs=DataVarAttrs(
-                    short_name="u80",
-                    long_name="U-component of wind (80 m above ground)",
-                    units="m/s",
-                    step_type="instant",
-                    standard_name="eastward_wind",
-                ),
-                internal_attrs=NoaaHrrrInternalAttrs(
-                    grib_element="UGRD",
-                    grib_description='80[m] HTGL="Specified height level above ground"',
-                    grib_index_level="80 m above ground",
-                    index_position=60,
-                    keep_mantissa_bits=6,
-                    hrrr_file_type="sfc",
-                ),
-            ),
-            NoaaHrrrDataVar(
-                name="wind_v_80m",
-                encoding=encoding_float32_default,
-                attrs=DataVarAttrs(
-                    short_name="v80",
-                    long_name="V-component of wind (80 m above ground)",
-                    units="m/s",
-                    step_type="instant",
-                    standard_name="northward_wind",
-                ),
-                internal_attrs=NoaaHrrrInternalAttrs(
-                    grib_element="VGRD",
-                    grib_description='80[m] HTGL="Specified height level above ground"',
-                    grib_index_level="80 m above ground",
-                    index_position=61,
-                    keep_mantissa_bits=6,
-                    hrrr_file_type="sfc",
-                ),
-            ),
-        ]
-
-    def _spatial_info(
-        self,
-    ) -> tuple[
-        tuple[int, int], tuple[float, float, float, float], tuple[float, float], str
-    ]:
-        """
-        Returns (shape, bounds, resolution, crs proj4 string).
-        Useful for deriving x, y and latitude, longitude coordinates.
-        See tests/noaa/hrrr/forecast_48_hour/template_config_test.py::test_spatial_info_matches_file
-        """
-        return (
-            (1059, 1799),
-            (
-                -2699020.142521929,
-                -1588806.152556665,
-                2697979.857478071,
-                1588193.847443335,
-            ),
-            (3000.0, -3000.0),
-            "+proj=lcc +lat_0=38.5 +lon_0=-97.5 +lat_1=38.5 +lat_2=38.5 +x_0=0 +y_0=0 +R=6371229 +units=m +no_defs=True",
-        )
+        return self.get_data_vars(encoding_float32_default)
```

Comment:
I wonder if `var_chunks`, `var_shards` and `var_encoding` should be defined as properties or methods on the TemplateConfig. That would allow you to avoid needing to know to call this `self.get_data_vars` and pass things through, since the parent `data_vars` could reference `self.var_encoding`